skyportal/handlers/api/photometric_series.py

```diff
@@ -0,0 +1,1764 @@
+              or for mosiaced CCD images where each tile has its own channel ID.
+              This allows multiple series to be saved with the same series name
+              but different channels, without violating the uniqueness constraint.
+              Series with different channels can have the same or different filters.
+              This field is entirely optional.
+            required: false
+          magref:
+            type: number
+            description: |
+              Reference magnitude for the photometric series.
+              This is used when the photometry is relative
+              (e.g., based on subtraction images) and the magnitude
+              of the object when it is not active is measured separately.
+              This would be the magnitude before/after a transient,
+              or the mean magnitude of a variable.
+              For absolute photometry this is left as None.
+            required: false
+          magref_unc:
+            type: number
+            description: uncertainty on the magref.
+            required: false
+          ra_unc:
+            type: number
+            description: uncertainty on the ra.
+            required: false
+          dec_unc:
+            type: number
+            description: uncertainty on the dec.
+            required: false
+          followup_request_id:
+            type: integer
+            description: |
+                ID of the followup request that generated this photometric series.
+                This is used to link the photometric series to the followup request
+                in the SkyPortal database.
+            required: false
+          assignment_id:
+            type: integer
+            description: |
+                ID of the assignment that generated this photometric series.
+                This is used to link the photometric series to the assignment
+                in the SkyPortal database.
+            required: false
+          time_stamp_alignment:
+            type: string
+            description: |
+              Specify when the time stamp for each measurement was taken
+              inside each exposure. Possible values are "start", "middle", "end".
+              This is optional, and defaults to "middle".
+            required: false
+          altdata:
+            type: object
+            description: |
+                Additional information to store in the photometric series.
+                This can be any valid JSON object, and will be stored
+                in the database as a JSON string.
+                This can hold various information that does not fit into
+                any of the other inputs, but will still be useful to keep track of.
+            required: false
+        """
+
+
+log = make_log('api/photometric_series')
+
+DEFAULT_SERIES_PER_PAGE = 100
+MAX_SERIES_PER_PAGE = 500
+
+
+def get_group_ids(data, user, session):
```

Comment:
Is this different from the photometry one? Can it just be imported from there?

Reply:
I've tried to do that, but the existing `get_group_ids` (and the streams one) are a little outdated, e.g., they don't get the session input and I didn't want to touch that code. I suppose we could merge them at some point.

Reply:
Ok makes sense, maybe open an issue so we take care of it?

Reply:
done